web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1223,11 +1223,16 @@ $(function() {
                     // location.href is something like foo.com/around?pc=abc-123,
                     // which we pass into fixmystreet.display.reports_list() as a fallback
                     // in case the list isn't already in the DOM.
+                    $('select[multiple] option').prop('selected', function() { return this.defaultSelected; }).trigger('change.multiselect');
                     fixmystreet.display.reports_list(location.href);
                 } else if ('reportId' in e.state) {
                     fixmystreet.display.report(e.state.reportPageUrl, e.state.reportId);
                 } else if ('newReportAtLonlat' in e.state) {
                     fixmystreet.display.begin_report(e.state.newReportAtLonlat, false);
+                } else if ('filter_change' in e.state) {
+                    $('#filter_categories').val(e.state.filter_change.filter_categories);
+                    $('#statuses').val(e.state.filter_change.statuses);
+                    $('select[multiple]').trigger('change.filters').trigger('change.multiselect');
```

Comment:
Sure (well, $('#filter_categories').add('#statuses') is about 10 times faster :) )